### PR TITLE
Personal Plan: 6GB Storage option

### PR DESCRIPTION
**Pull request by rodrigoi**
Adds a 6GB storage option for the Personal Plan behind a feature flag.

cc: @roundhill

### CODE REVIEW

File: client/lib/plans/constants.js [hunk 1]
```diff
@@ -518,6 +519,17 @@ export const FEATURES_LIST = {
...
 	},
 
+	[ FEATURE_6GB_STORAGE ]: {
+		getSlug: () => FEATURE_6GB_STORAGE,
+		getTitle: () => i18n.translate( '{{strong}}6GB{{/strong}} Storage Space', {
+			components: {
+				strong: <strong />
+			}
+		} ),		getDescription: () => i18n.translate(
+			'Storage space for adding images and documents to your website.'
```
Comment (roundhill) [inline client/lib/plans/constants.js]: Let's copy what the other space upgrades use for the description:

`With increased storage space you'll be able to upload more images, videos, audio, and documents to your website.`

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -111,15 +112,15 @@ export const PLANS_LIST = {
 					strong: <strong className="plan-features__targeted-description-heading" />
 				}
 			} ),
-		getFeatures: () => [
+		getFeatures: () => compact ( [
```
Comment (gwwar) [inline client/lib/plans/constants.js]: When would this array have a falsey value?

File: client/lib/plans/constants.js [hunk 3]
```diff
@@ -526,7 +540,7 @@ export const FEATURES_LIST = {
 			}
 		} ),
 		getDescription: () => i18n.translate(
-			"With increased storage space you'll be able to upload " +
+			'With increased storage space you\'ll be able to upload ' +
```
Comment (gwwar) [inline client/lib/plans/constants.js]: Just fyi that both forms are currently accepted for escaping `'`.

**Comment by gwwar** [comment]
👍  Looks good. I assume we're queuing this up with the backend patch since storage can be checked in a few different areas like in the media modal or wp-admin
screen shot 2016-11-29 at 4 37 17 pm

**Comment by rodrigoi** [comment]
@gwwar you assume correctly. Backend diffs to add the new storage product and to upgrade current users is coming soon.

**Comment by roundhill** [comment]
Looks good to me, :shipit:

**Pull request merged**